### PR TITLE
fix(ci): avoid create comment workflows failing when required worfklows did not run

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area CI

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

Avoids all the failings like: https://github.com/falcosecurity/libs/actions/workflows/create-comment-kernel-testing.yml

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by FedeDP** [comment]
Wip while i test it.

File: .github/workflows/create-comment-kernel-testing.yml [hunk 1]
```diff
@@ -36,7 +40,10 @@ jobs:
             fs.writeFileSync('${{github.workspace}}/pr.zip', Buffer.from(download.data));
 
       - name: 'Unpack artifact'
-        run: unzip pr.zip
+        run:
+          if [ -f pr.zip ]; then
```
Comment (FedeDP) [inline .github/workflows/create-comment-kernel-testing.yml:44]: If no `pr.zip` was downloaded by previous step.

File: .github/workflows/create-comment-kernel-testing.yml [hunk 2]
```diff
@@ -45,6 +52,9 @@ jobs:
           # Taken from https://github.com/actions/github-script/blob/main/.github/workflows/pull-request-test.yml
           script: |
             var fs = require('fs');
+            if (!fs.existsSync('./NR')) {
```
Comment (FedeDP) [inline .github/workflows/create-comment-kernel-testing.yml:55]: If there is no `NR` file, then no artifact was downloaded. Leave.

**Review by Andreagit97 — APPROVED** [review]
/approve

**Review by leogr — APPROVED** [review]

**Pull request merged**